swagger.json

```
@@ -4041,6 +4044,22 @@
       },
       "x-go-package": "github.com/kiali/kiali/models"
     },
+    "ResponseFlags": {
+      "description": "\"200\" : {\n\"-\"     : 80.0,\n\"DC\"    : 10.0,\n\"FI,FD\" : 10.0\n}, ...",
```

The values are strings also. So it should be:

```suggestion
      "description": "\"200\" : {\n\"-\"     : \"80.0\",\n\"DC\"    : \"10.0\",\n\"FI,FD\" : \"10.0\"\n}, ...",
```

---

a new swagger-gen should fix that now that I updated the source file.